src/plugins/meta.ts

```diff
@@ -1,7 +1,132 @@
+import fetch from 'node-fetch';
+import { EntityManager } from 'typeorm';
+
 import { FastifyPluginAsync } from 'fastify';
 
+import { SearchService } from '../services/item/plugins/published/plugins/search/service';
+import { EMBEDDED_LINK_ITEM_IFRAMELY_HREF_ORIGIN, ETHERPAD_URL } from '../utils/config';
+
+class BaseStatus {
+  private isOk: boolean;
```

Comment:
```suggestion
  private isHealthy: boolean;
```

Reply:
Or do we actually want a `status` with some enum `Healthy`, `Busy`, `Unresponsive` (almost random examples)?

Reply:
Yes maybe have a generic `ServiceStatus` class, with a `status` enum and eventually a computed (during `format`) `isOk` boolean depending on the `status` value.